docs/package.json

```diff
@@ -10,20 +10,28 @@
 		"astro": "astro"
 	},
 	"dependencies": {
-		"astro": "^2.0.15",
-		"preact": "^10.7.3",
-		"react": "^18.1.0",
-		"react-dom": "^18.1.0",
-		"@astrojs/react": "^2.0.2",
-		"@astrojs/preact": "^2.0.3",
 		"@algolia/client-search": "^4.13.1",
+		"@astrojs/mdx": "^0.17.2",
+		"@astrojs/preact": "^2.0.3",
+		"@astrojs/react": "^2.0.2",
 		"@docsearch/css": "^3.1.0",
 		"@docsearch/react": "^3.1.0",
-		"@types/react": "^17.0.45",
```

Comment:
These @types/* can go to  devDependencies

Reply:
moved